### PR TITLE
Bfix(#490): Fixes distance between found items in accounts search

### PR DESCRIPTION
**Pull request by sopu**
screenshot from 2016-03-16 17 46 27
Partial fix to #490. 
Please don't merge right now!

### CODE REVIEW

**Comment by Orbiter** [comment]
What is the status here, why "Please don't merge right now!"?

**Comment by sopu** [comment]
I have added patches for all parts of #490. @sudheesh001 please see!

File: app/js/controllers/search.js [hunk 1]
```diff
@@ -186,8 +186,15 @@ controllersModule.controller('SearchCtrl', ['$window', '$stateParams', '$rootSco
...
                 vm.peopleSearch = true;
-                vm.showMap = false;
+                vm.showMap = false;              
             }, function() {});
+            vm.accountsPretty.forEach(function(ele) {
+                var result = $filter('filter')($rootScope.userTopology.followers, {screen_name : ele.screen_name}, true);
+                if(result.length) {
+                    ele.friend = true;
+                }
+                else ele.friend = false;
```
Comment (sudheesh001) [inline app/js/controllers/search.js]: Codacy Issue found: [Expected { after 'else'.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2624679912/issues/source?bid=3288768&fileBranchId=3288802#l196)

Comment (sudheesh001) [inline app/js/controllers/search.js]: Codacy Issue found: [Expected '{' and instead saw 'ele'.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2624679912/issues/source?bid=3288768&fileBranchId=3288802#l196)

File: app/js/controllers/search.js [hunk 2]
```diff
@@ -186,8 +186,15 @@ controllersModule.controller('SearchCtrl', ['$window', '$stateParams', '$rootSco
             SearchService.retrieveMultipleImg(vm.accounts).then(function(data) {
                 vm.accountsPretty = data.users;
                 vm.peopleSearch = true;
-                vm.showMap = false;
+                vm.showMap = false;              
             }, function() {});
+            vm.accountsPretty.forEach(function(ele) {
+                var result = $filter('filter')($rootScope.userTopology.followers, {screen_name : ele.screen_name}, true);
```
Comment (sudheesh001) [inline app/js/controllers/search.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2624679912/issues/source?bid=3288768&fileBranchId=3288802#l192)

Comment (sudheesh001) [inline app/js/controllers/search.js]: Codacy Issue found: [Identifier 'screen_name' is not in camel case.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2624679912/issues/source?bid=3288768&fileBranchId=3288802#l192)

File: app/js/controllers/search.js [hunk 3]
```diff
@@ -186,8 +186,15 @@ controllersModule.controller('SearchCtrl', ['$window', '$stateParams', '$rootSco
             SearchService.retrieveMultipleImg(vm.accounts).then(function(data) {
                 vm.accountsPretty = data.users;
                 vm.peopleSearch = true;
-                vm.showMap = false;
+                vm.showMap = false;
```
Comment (sudheesh001) [inline app/js/controllers/search.js]: Why the extra whitespaces here ?

File: app/views/account.html [hunk 4]
```diff
@@ -1,18 +1,23 @@
-
 <div ng-attr-id="{{data.id_str}}" class="home-user-info">
     <div class="blue-background-placeholder">
         <a ng-href="{{'./search?q=from:' + data.screen_name}}" ><img ng-src="{{data.profile_banner_url }}" alt="user-banner-photo" ng-show="data.profile_banner_url" fallback-src="/images/anon_400x400.png"></a>
     </div>
     <div class="home-user-info-content">
         <div class="top-content">
-            <a ng-href="{{'./search?q=from:' + data.screen_name}}"><img fallback-src="/images/anon_200x200.png" ng-src="{{data.profile_image_url_https | homePageImage }}"></a>
+            <div class="account-image-and-following-text">
```
Comment (sudheesh001) [inline app/views/account.html]: Where's the css for this class `.account-image-and-following-text` ?

Comment (sopu) [inline app/views/account.html]: Just added to combine things in a div. After "following" button is introduced. @sudheesh001 
No CSS as of yet. Can remove class.

**Pull request merged**